src/aap_eda/settings/default.py

```diff
@@ -806,7 +797,11 @@ def get_rulebook_process_log_level() -> RulebookProcessLogLevel:
     f"port={DATABASES['default']['PORT']} "
     f"dbname={DATABASES['default']['NAME']} "
     f"user={DATABASES['default']['USER']} "
-    f"password={DATABASES['default']['PASSWORD']}"
+    f"password={DATABASES['default']['PASSWORD']} "
+    f"sslmode={settings.get('PGSSLMODE', default='prefer')} "
```

Comment:
I'm not sure if this is the right approach to handle the settings. We should be honoring `DATABASES[alias][options]`
So we could be able to use or override any existing alias defined. This is important also to support a different database for pg_notify. And that would be in line with AWX too.
Even if we use a global unique setting for `PGSSLCERT`and `PGSSLKEY` and so on, those variables should have a prefix or suffix to clearly indicate where are used and avoid ambiguities.